R/vdiagram-data.R

```diff
@@ -97,3 +97,21 @@ defaultHistFlags <- function(histFlag){
   
   return(histFlag);
 }
+
+#' Create Control Condition String
+#' 
+#' @description Creates a comma-separated string of the control conditions
+#' @param controlConditions The vector of control conditions to create the string from
+createControlConditionsString <- function(controlConditions){
+  returnString <- ""
+  
+  if(!isEmptyOrBlank(controlConditions)){
+    displayValues <- controlConditions[['name']]
+    
+    returnString <- paste(sapply(displayValues, function(condition){
+      return(sapply(strsplit(condition, "\\_"), function(part) paste(camelCase(part), collapse=" ")))
```

Comment:
Display values is a vector of all of the excluded control condition strings formated as `UPPERCASE_WITH_UNDERSCORES`. The outer sapply is to process all of the strings, the inner sapply is to split the current string at the underscores and then camel case each underscore.